src/main/java/models/jwtToken/DecodedTokenHeader.java

```diff
@@ -0,0 +1,43 @@
+package models.jwtToken;
+
+import java.util.Objects;
+
+public class DecodedTokenHeader {
+    private String alg;
+    private String typ;
+
+    public static class Builder {
+        private DecodedTokenHeader decodedTokenHeader;
+
+        public Builder() {
+            decodedTokenHeader = new DecodedTokenHeader();
+        }
+
+        public Builder addAlg(String alg) {
+            decodedTokenHeader.alg = alg;
+            return this;
+        }
+
+        public Builder addTyp(String typ) {
+            decodedTokenHeader.typ = typ;
+            return this;
+        }
+
+        public DecodedTokenHeader build() {
+            return decodedTokenHeader;
+        }
+    }
+
+    @Override
+    public boolean equals(Object o) {
+        if (this == o) return true;
```

Comment:
I need an explanation here. It is totally has no sense for me

Reply:
Removed.